multinet/validation/__init__.py

```diff
@@ -1,14 +1,14 @@
 """Validation errors for various multinet processes."""
 from pydantic import BaseModel
-from typing import List, Dict
+from typing import List, Dict, Any
 
 
 class ValidationFailure(BaseModel):
     """Base class for any validation errors."""
 
-    def validation_dict(self) -> Dict:
-        """Return a dict representation of the Validation Failure."""
-        return {**self.dict(), "type": self.schema()["title"]}
+    def dict(self, **kwargs: Any) -> Dict:  # noqa: A003
```

Comment:
Is this ok to do? What bad things could possibly happen because of overloading a built-in like this?

Reply:
Well this isn't an overload of a built-in, since this is a class method, and the built-in `dict` is just a function. I think flake8 just sees a function named `dict` and complains about it.

Reply:
Oh, I see. Seems like a bug in `flake8`.